setDefaultsPTB.m

```diff
@@ -0,0 +1,37 @@
+function cfg = setDefaultsPTB(cfg)
```

Comment:
overall this is a good idea and I see here the importance of providing the experiment template with crystal clear explanation of which params are for what and how to tweak them.

Reply:
... so that people like Ceren would not bother you ever again :)

Reply:
haha no that is impossible :)

Reply:
you know that right!